lib/modules.nix

```diff
@@ -57,6 +57,31 @@ in
       }
       // extraSpecialArgs;
     };
+
+  /**
+    Build a Nixvim package.
+
+    This is a thin wrapper around `<nixvim-configuration>.config.build.package`.
+
+    # Inputs
+
+    `input`
+    : One of:
+      1. Arguments for `evalNixvim`.
+      2. A Nixvim configuration, as produced by `evalNixvim`.
+      3. A Nixvim package, as produced by `buildNixvim`.
+
+    # Output
+
+    An installable Nixvim package.
+  */
+  buildNixvim = lib.mirrorFunctionArgs self.modules.evalNixvim (
```

Comment:
Is the `buildNixvim` mentionned in the docstring the same as this `buildNixvim`?
Otherwise, wouldn't you feed `buildNixvim` with _a Nixvim package, as produced by `buildNixvim`_?